collect/central_collector.go

```diff
@@ -668,6 +668,17 @@ func (c *CentralCollector) cleanupTraces(ctx context.Context) {
 	if len(ids) == 0 {
 		return
 	}
+	for _, t := range ids {
+		trace := c.SpanCache.Get(t)
+		if trace == nil {
+			continue
+		}
+		data := make(map[string]interface{}, 0)
+		data["trace_id"] = t
+		data["arrival_time"] = trace.ArrivalTime
+		data["span_coutn"] = len(trace.GetSpans())
```

Comment:
spelling of `span_count`

And should these be `meta` fields?

Reply:
Ooops, this is my debugging code lol